### ISSUE TITLE
Feature Request: Make full page text searchable

### ISSUE THREAD

**Issue by nizdobs** [babyman/quick-tabs-chrome-extension#105]
I'd love to be able to search against the full html text in an open tab. Is there any way to do this in the product today? Do you have thoughts to make this feature available in the future?

Thanks, 

Niz

**Comment by babyman** [comment]
I like this idea but I'm not sure how it could be implemented without killing the extensions performance...

**Comment by fcherman** [comment]
Whoa, wouldn't that be lovely?

**Comment by crossan007** [comment]
Yes Please!

**Comment by crossan007** [comment]
I'm 100% new to extension development, but maybe starting service workers with async callbacks on matches?  high CPU usage, but should keep UX responsive...

**Comment by crossan007** [comment]
https://www.html5rocks.com/en/tutorials/workers/basics/  ?

**Comment by JackYoungblood** [comment]
> I like this idea but I'm not sure how it could be implemented without killing the extensions performance...

Perhaps make it so that full page text search is not the default extension behavior but could be invoked in a different way. A user would have to accept that searching this way would be slower.

**Comment by crossan007** [comment]
It looks like this feature is coming to Chrome native: https://www.howtogeek.com/704212/how-to-search-open-tabs-on-google-chrome/

**Comment by fcherman** [comment]
> It looks like this feature is coming to Chrome native: https://www.howtogeek.com/704212/how-to-search-open-tabs-on-google-chrome/

Nope. I have the tab search feature in Chrome Canary. It searches only page titles and urls; it does not search the full text of the page they represent. Just to be sure, I searched my tabs for the word "lovely" (as you can see, it appears on this page in my comment from June 20, 2016. My search turned up 0 results.
